### ISSUE TITLE
Add pyppeteer support

### ISSUE THREAD

**Issue by HenryGetz** [holzschu/a-shell#780]
Just as simple as the title-- adding working support for the https://github.com/pyppeteer/pyppeteer library

**Comment by bbb651** [comment]
It's definitely impossible to run because it targets chromium specifically and you only have WebKit (Safari) on iOS (well you can have 3rd party browser engines in the EU, but it's a tremendous amount of work if it's even possible, and it's unreasonable to bundle chromium with a-shell which would quadruple it's size and restrict it to the EU), and it's deprecated and unmaintained regardless. Something like https://github.com/appium/appium-safari-driver seems more reasonable because it utilizes the WebKit webview and can run on iOS, I'm not sure how much effort it'll be to get it to run in a-shell

**Comment by HenryGetz** [comment]
@bbb651 yeah that's fair. I just kind of hoped that wasn't how it worked (though of course that's how it works in brew and everywhere else) yet didn't research it, but that makes sense. The workflows this would allow for though would be insane, that would be dope if we could get that safari project running
